workers/unity/Assets/Playground/Scripts/Worker/WorkerConnectorBase.cs

```diff
@@ -35,11 +35,11 @@ private void OnDestroy()
             Dispose();
         }
 
-        public async Task Connect(string workerType, ILogDispatcher logger)
+        protected async Task Connect(string workerType, ILogDispatcher logger)
```

Comment:
This is meant to be public. We want other things to be able to call connect on the derived class. We currently don't because we do things in start but that won't work in all cases.

Reply:
Actually we haven't made this virtual yet. And this meets are use cases for now.
In that case I retract the above. Unless we want to rejig all of this protected is fine.

Reply:
Actually thinking about this again, does it make any sense for this to be protected and the class not be abstract. This makes it mostly useless if you don't inherit from it.